packages/theme-dark-extension/style/variables.css

```diff
@@ -374,4 +374,15 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-search-toggle-off-opacity: 0.5;
   --jp-search-toggle-hover-opacity: 0.75;
   --jp-search-toggle-on-opacity: 1;
+
+  /* scrollbar related styles. Supports every browser except Edge.
+   * On Firefox, only the color settings are respected.
+   */
+
+  --jp-scrollbar-background-color: #3f4244;
```

Comment:
Can you add a note to say where this color came from - otherwise looks arbitrary.

Reply:
/* Based on the JetBrains Darcula theme */

All of this CSS is copied directly from my [Darcula theme](https://github.com/telamonian/theme-darcula). I'm no designer, so when I couldn't find an authoritative reference for Material Dark, I figured it was safer to just leave the colors alone (they do look pretty good). If you want to point me to a spec (or just an example of a good Material Dark theme in action) I'd be happy to change these.

Reply:
note added